CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -392,6 +385,15 @@ public int Initialise(CVKDevice cvkDevice) {
         int ret = VK_SUCCESS;
         
         this.cvkDevice = cvkDevice;
+
+        // Initialise push constants to identity mtx
+        pushConstants = memAlloc(VertexUniformBufferObject.SIZEOF);
+        for (int iRow = 0; iRow < 4; ++iRow) {
+            for (int iCol = 0; iCol < 4; ++iCol) {
+                pushConstants.putFloat(IDENTITY_44F.get(iRow, iCol));
```

Comment:
Do you think we should move that identity constant somewhere shared?  Not a big deal.

Reply:
I think it is ok as is (the identity mtx is defined in the matrix class Matrix44f.identity();), so either way you would have to import it (both 1 liners).

Reply:
Just a memory thing, we are constructing 64Bytes of floats + whatever java overhead each time we have an identity mtx.  Not a big deal.

Reply:
Leaving this one as is for this change